mappings/net/minecraft/server/world/ThreadedAnvilChunkStorage.mapping

```diff
@@ -164,6 +164,10 @@ CLASS net/minecraft/class_3898 net/minecraft/server/world/ThreadedAnvilChunkStor
 	METHOD method_34869 getChebyshevDistance (Lnet/minecraft/class_1923;Lnet/minecraft/class_1297;)I
 		ARG 0 chunkPos
 		ARG 1 entity
+	METHOD method_37831 (Lnet/minecraft/class_1923;Lnet/minecraft/class_3222;)Z
+		ARG 1 player
+	METHOD method_37832 getMobSpawnablePlayers (Lnet/minecraft/class_1923;)Ljava/util/List;
```

Comment:
This return `List<EntityLike>`, though it only returns players in the current implementation. Should it be named `Players` or `Entities`/ `Targets`?

Reply:
I don't think it makes sense for this method to ever return non-players. Random mob spawn only happens around players, and not other mobs - and this method uses operations only applicable to players, such as checking gamemode. Maybe in the future it may change, but when that happens it'll probably fail in the matching process and become a new method.

Reply:
Yeah, this is totally fine. We can always revisit this later; maybe they just limited the return type to prevent access to unnecessary functionalities.